### ISSUE TITLE
Update accessibility checklist

### ISSUE THREAD

**Issue by blackbaud-johnly** [blackbaud/skyux2-docs#922]
Update https://developer.blackbaud.com/skyux/learn/accessibility/design/checklist to use SKY UX checkboxes.
